docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -486,18 +484,15 @@ Development builds typically display an onboarding welcome screen when an app is
 An example of such a Detox test is shown below. Full example code is available on the [eas-tests-example](https://github.com/expo/eas-tests-example) repository.
```

Comment:
Maybe add a note that https://github.com/expo/eas-tests-example/blob/main/eas-hooks/eas-build-on-success.sh has to updated for this to work.